admin/js/common.js

```diff
@@ -649,7 +649,7 @@ function qtranxj_ce(tagName, props, pNode, isFirst) {
 
         var updateTinyMCE = function (h) {
             text = h.contentField.value;
-            if (h.wpautop && window.switchEditors) {
+            if (1) {
```

Comment:
You are removing the if-check without removing it.
About the 2 conditions:
- why do you remove the check h.wpautop? 
- why do you remove the check on window.switchEditors? It is still used in your code.

Reply:
In "classic" WYSIWYG editor with ACF and qtranslate-xt wordpress does not use wpautop().
And it makes paragraphs and new line deleted. during opening editor.

This make wpautop works anyway

Reply:
I'm not sure about these changes, more investigation and testing is needed to understand this better. I would also like WP to fix the issue with Firefox #931.

Reply:
Not that i don't want to merge the fix, but the change would need to be reworked a bit. If the if-check is not necessary it should be completely removed but i feel it was there for good reasons.

Reply:
wpautop is disabled in the qTranslate PHP code but not in the JS code, see `updateTinyMCE` in `admin/common.js` and all the comments from 3.2.9.8 "ugly trick" about `updateTinyMCEonInit`.
I don't think we can disable it like this. Maybe we need a special case for ACF WYSIWYG but I would not do it in all cases.